utils/useLocalStorage.ts

```diff
@@ -0,0 +1,53 @@
+import { useCallback, useEffect, useState } from 'react'
```

Comment:
Can we use https://github.com/streamich/react-use/blob/master/docs/useLocalStorage.md from react-use? It is available in our dependencies

Reply:
I don't see why not, I can swap it in and see how it plays with the existing usage of the hook

Reply:
awesome!

Reply:
Have to install `react-use` to use that hook and in doing so includes a lot of other things

Swapping out the hook causes some weird UI behavior --
<img width="1021" alt="image" src="https://user-images.githubusercontent.com/3385743/176573537-17c0b180-25b8-4892-a055-71e58a989925.png">

The only code change that caused the above..
<img width="439" alt="image" src="https://user-images.githubusercontent.com/3385743/176575546-96aad533-0dd1-4b63-b72a-65c0d083571e.png">